runtime/src/integration/mod.rs

```diff
@@ -223,6 +191,13 @@ pub async fn with_timeout<T: Future>(future: T, duration: Duration) -> T::Output
     timeout(duration, future).await.expect("timeout")
 }
 
+pub async fn start_chain() -> std::io::Result<Child> {
+    let command = Command::new("sh").arg("../scripts/run_parachain_node.sh").spawn();
+    // Time to start the parachain image, so RPC request can be handle
+    tokio::time::sleep(Duration::from_secs(2)).await;
```

Comment:
I don't like sleepers such as this, we already had a big problem in the Typescript library with these. Where possible I would prefer to await some expected state such the result of making a successful RPC connection.

Reply:
I think `from_url_with_retry` should already do that, if there is a specific test case that fails maybe reorder the setup logic.

Reply:
Removed the sleep duration and stopped the old parachain instance before running the script in the background thread.